monitoring/messagesigning/mockuss_well_known_public_key.py

```diff
@@ -0,0 +1,20 @@
+import os
+import flask
+from loguru import logger
+from config import Config
+
+webapp = flask.Flask(__name__)
+
+
+@webapp.route("/mock/scd/.well-known/uas-traffic-management/mock_pub.der", methods=["GET"])
```

Comment:
Is the well-known endpoint service-specific or a general feature USSs will need to enable for all services including Remote ID for instance?
In any case, this implementation will need to move to the [`mock_uss`](https://github.com/interuss/dss/tree/master/monitoring/mock_uss) module, either in a new service folder or in [mock_uss/routes.py](https://github.com/interuss/dss/blob/master/monitoring/mock_uss/routes.py) if this endpoint is unique for a USS.
Please note that the activation of this endpoint should be subject to a configuration flag disabled by default.

Reply:
mock_uss will sign its responses according to the latest [spec](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-message-signatures-11). This endpoint provides the public key needed to validate these signatures. Other USSs will not necessarily need this endpoint, so long as when they sign their requests/responses, in their `x5u` field, they provide a publicly accessible endpoint that other USSs can use to validate their signatures.